### ISSUE TITLE
Vote weight - Distribute votes to different options

### ISSUE THREAD

**Issue by eiclu** [OpenSlides/OpenSlides#6587]
**Is your feature request related to a problem? Please describe.**
Right now when voting for a motion, only one option can be chosen, even if one has a vote weight of more than one.

**Describe the solution you'd like**
Give admins an option to allow accounts with a higher vote weight than one to distribute their votes on multiple options.

**Describe alternatives you've considered**
Right now you can kind of implement this, by giving the affected people multiple accounts, but this is a hassle for all involved.

**Additional context**
In our organization, subdivisions have two votes each. They decide within their group how these two votes should be distributed for each motion. If a motion is controversial, these two votes could differ, e.g. one yes and one no. 
The subdivisions have one account each to vote. Right now we couldn't reflect this voting method from within OpenSlides.

**Comment by eiclu** [comment]
Yeah, you're right that with floating-point weights this doesn't seem to work properly.
As a simple workaround the voting weights could be restricted to integer numbers when the feature is enabled

**Comment by jsangmeister** [comment]
This would require further changes in the voting methods and is unfortunately not feasible right now. As a practical workaround, you could introduce vote delegation between the two accounts you have for each subdivision, so one account can vote for both of them.

**Comment by Elblinator** [comment]
We do not plan to implement this at all
